### ISSUE TITLE
Warning when running pylint in pyenv (WARNING: no pylint version could be extracted from)

### ISSUE THREAD

**Issue by duydao** [SublimeLinter/SublimeLinter-pylint#4]
SublimeLinter-pylint checks the third line of the pylint executable for the version. Unfortunately, this won't work well with pyenv since the executable is just a wrapper for pylint itself.

**expected by SublimeLinter**

```
...
# EASY-INSTALL-ENTRY-SCRIPT: 'pylint==1.1.0','console_scripts','pylint'
__requires__ = 'pylint==1.1.0'
...
```

**actual content `~/.pyenv/shims/pylint`**

```
#!/usr/bin/env bash
set -e
[ -n "$PYENV_DEBUG" ] && set -x

program="${0##*/}"
...
```

we can avoid this warning by adding the actual path of pylint to `SublimeLinter.sublime-settings`:

```
...
"paths": {
    "linux": [],
    "osx": [
        "~/.pyenv/versions/3.3.3/bin"
    ],
    "windows": []
},
...
```

Why do I get this warning and does it have any side effects on SublimeLinter-pylint?
